.clj-kondo/funcool/promesa/config.edn

```diff
@@ -0,0 +1,9 @@
+{:lint-as {promesa.core/->          clojure.core/->
```

Comment:
Just to confirm, are you going to remove code changes related to `promesa`? Since the lib won't be added in this PR.